packages/ui/src/lib/toaster/Toaster.stories.svelte

```diff
@@ -0,0 +1,140 @@
+<script lang="ts">
+	import { Meta, Story } from '@storybook/addon-svelte-csf';
+	import Button from '../button/Button.svelte';
+
+	import Toaster from './Toaster.svelte';
+	import { newToastMessage } from './toaster';
+	import { ToastType } from './types';
+
+	import typeDocs from './types?raw';
+
+	const toastNotice = () => {
+		newToastMessage('A notice!').post();
+	};
+
+	const toastSuccess = () => {
+		newToastMessage('A success!', { type: ToastType.Success }).post();
+	};
+
+	const toastWarning = () => {
+		newToastMessage('A warning!', { type: ToastType.Warning }).post();
+	};
+
+	const toastError = () => {
+		newToastMessage('An error!', { type: ToastType.Error }).post();
+	};
+
+	const longLivedMessage = newToastMessage(
+		'These messages decay and disappear after five seconds. You can set a' +
```

Comment:
This is slightly confusing, because the message that states this is long-lived.


Cosndier editing to something like "By default, messages disappear after 5 seconds. You can set a longer timeToLive (this message has it set to 5000) ..."